pkg/dogstatsd/server_test.go

```diff
@@ -581,8 +581,8 @@ dogstatsd_mapper_profiles:
 			},
 			expectedSamples: []MetricSample{
 				{Name: "test.job.duration", Tags: []string{"job_type:my_job_type", "job_name:my_job_name"}, Mtype: metrics.GaugeType, Value: 666.0},
-				{Name: "test.job.duration.my_job_type.my_job_name", Tags: []string{"some:tag"}, Mtype: metrics.GaugeType, Value: 666.0},
-				{Name: "test.job.duration.my_job_type.my_job_name", Tags: []string{"some:tag", "more:tags"}, Mtype: metrics.GaugeType, Value: 666.0},
+				{Name: "test.job.duration", Tags: []string{"job_type:my_job_type", "job_name:my_job_name", "some:tag"}, Mtype: metrics.GaugeType, Value: 666.0},
```

Comment:
Would upgrading the agent cause any impact on customers monitoring with the tag included in the metric name?

Reply:
@ChristineTChen Yes, that's a possibility, but quite unlikely.

If a user sends 1) metrics with tags and have 2) mapping rules (prefix **and** match rules) that match that metric the DSD Mapper will be effective (not the case before).

But this case is unlikely. For it to happen user need:
- Metrics that match the prefix of the DSD Mapper profile.
- Metrics that have tags
- Metrics match one of the mapping rules
- User do **not** want some metrics that has been matched by the mapping rules to be mapped.